Sources/FluentKit/Migration/Migrator.swift

```diff
@@ -246,13 +246,17 @@ private final class DatabaseMigrator {
     // MARK: Private
 
     private func prepare(_ migration: Migration, batch: Int) -> EventLoopFuture<Void> {
+        self.database.logger.info("[Migrator] Starting prepare of migration '\(migration.name)'")
```

Comment:
Actually I know I said yes to this but let's move the migration name into the logger metadata instead